spyder/plugins/editor/panels/scrollflag.py

```diff
@@ -229,8 +208,7 @@ def compute_flag_ypos(block):
             painter.setPen(self._edgecolors[flag_type])
             for block_number in dict_flag_lists[flag_type]:
                 # Don't paint local flags outside of the window
-                if paint_local and not (
-                        min_line <= block_number + 1 <= max_line):
+                if paint_local:
```

Comment:
I think we should remove `paint_local` because it seems to be doing very little now.